db/migrate/20230203074700_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, references: :users, null: false, foreign_key: { to_table: :users }
```

Comment:
- [ ] Kindly remove the second reference as this is achieved by the foreign key constraint which points to the user as the `author`, leaving your migration file looking like the following
`t.references :author, foreign_key: { to_table: 'users' }`